eZ/Publish/SPI/Persistence/Content/Handler.php

```diff
@@ -258,4 +258,13 @@ public function loadReverseRelations($destinationContentId, $type = null);
      * @return \eZ\Publish\SPI\Persistence\Content The published Content
      */
     public function publish($contentId, $versionNo, MetadataUpdateStruct $metaDataUpdateStruct);
+
+    /**
+     * Remove the specified translation from all the Versions of a Content Object.
+     *
+     * @param int $contentId
+     * @param string $languageCode language code of the translation
+     * @param string $mainLanguageCode Content Object main language code
```

Comment:
Kind of agree, I wanted to avoid loading it once again in the persistence layer.
Removed this parameter in 4f81f18.
In ef783ff, there's an example how we can do it using nested `select` in `update` query.